content/new-author-metadata.yaml

```diff
@@ -47,3 +47,9 @@ author_info:
     email: smb310@georgetown.edu
     affiliations: Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, DC, USA
     funders: NIH R21CA220398, NIH P30CA051008
+  - github: alexandari
+    name: Amr Mohamed Alexandari
+    initials: AMA
+    orcid: 0000-0001-8655-8109
+    email: amr.alexandari@gmail.com
+    affiliations: Department of Computer Science, Stanford University
```

Comment:
#767 improved the affiliation handling so this line will have to be updated to:
```
    affiliations:
      - Department of Computer Science, Stanford University, Stanford, CA
```

Please merge with master to update the other authors' information as well and resolve the conflict.